src/Spryker/Client/RabbitMq/RabbitMqConfig.php

```diff
@@ -276,4 +276,12 @@ public function isDynamicStoreEnabled(): bool
     {
         return (bool)getenv('SPRYKER_DYNAMIC_STORE_MODE');
     }
+
+    /**
+     * @return int
+     */
+    protected function getHeartBeat(): int
+    {
+        return (int)$this->get(RabbitMqEnv::RABBITMQ_HEART_BEAT_SECONDS, 0);
```

Comment:
I know!
static::AMQP_STREAM_CONNECTION_HEART_BEAT - should be a default value, no 0.
Then it doesn't look like a BC break.

Reply:
good idea

Reply:
done